### PR TITLE
feat: add table of cli flags to getting started guide

### LINKED ISSUE #3023: Add Table of CLI Flags, Environment Variables, and Descriptions to Getting Started Guide
**What problem would you like to solve? Please describe:**
Currently, the Getting Started guide for GoAlert does not include a comprehensive table that lists all the command line flags, corresponding environment variables, and descriptions for the project.

**Describe the solution you'd like:**
I propose adding a table to the Getting Started guide, which includes:

1. All available command line flags
2. Corresponding environment variables for each flag
3. Clear and concise descriptions for each flag and variable

This table will make it easier for new users to understand and utilize GoAlert's configurable options and help new users get started quickly and efficiently.

**Additional context:**
This feature request aims to improve the documentation and user experience for new users of the GoAlert project.

### PR DESCRIPTION
**Pull request by allending313**
- [x] Identified the issue which this PR solves.
- [x] Read the [**CONTRIBUTING**](https://github.com/target/goalert/blob/master/CONTRIBUTING.md) document.
- [x] Code builds clean without any errors or warnings.
- [x] Added appropriate tests for any new functionality.
- [x] All new and existing tests passed.
- [x] Added comments in the code, where necessary.
- [x] Ran `make check` to catch common errors. Fixed any that came up.

**Description:**
Adds table to the getting started markdown with:
- All available command line flags
- Corresponding environment variables for each flag
- Clear and concise descriptions for each flag and variable

**Which issue(s) this PR fixes:**
Fixes #3023 

**Screenshots:**
Screenshot 2023-05-25 at 11 03 40 AM

### CODE REVIEW

**Comment by mastercactapus** [comment]
Looks great!

There are a few that are the main ones to be used, can we move those to the top?
- `--db-url` and `--db-url-next`
- `--public-url`
- `--data-encryption-key` and `--data-encryption-key-old`

Also because those are listed as the primary ones by this guide:
> To run GoAlert, you can start the binary directly, or from a container image. You will need to specify the `--db-url`, `--public-url`, and `--data-encryption-key` you plan to use.

File: docs/getting-started.md [hunk 1]
```diff
@@ -228,3 +228,52 @@ Twilio trial account limitations (if you decide to upgrade your Twilio account t
 
 - SMS: The message "Sent from your Twilio trial account" is prepended to all SMS messages
 - Voice: "You have a trial account..." verbal message before GoAlert message.
+
+
+### CLI Flags
+
+Additional options are available for running GoAlert in the form of CLI flags. Their corresponding enviornment variable names are listed as well.
```
Comment (m17ch) [inline docs/getting-started.md:235]: `enviornment` -> `environment`

Comment (allending313) [reply docs/getting-started.md:235]: ahhh, good catch

**Review by andrewbenington — DISMISSED** [review]
LGTM!

**Comment by mastercactapus** [comment]
I like the bold, but maybe leave out `--db-url-next` and `--data-encryption-key-old` from that style -- it also may be nice to add to the description of each (to flag and table) something like "only use this when performing a switchover" and "only necessary when changing --data-encryption-key" respectively.

That way, the bold flags are the ones you need to use always, and the additional context will prevent confusion for new users.

**Review by mastercactapus — APPROVED** [review]

**Review by andrewbenington — APPROVED** [review]
👍

**Pull request merged**